ffvalidate/README.md

```diff
@@ -1,7 +1,7 @@
 # ffvalidate
 
-Identifies file formats in the given path and validates them against a
-list of allowed formats.
+Identifies file formats in the given path and validates them against a list of
```

Comment:
Identifies the file format of the files at the given path, recursively walking any sub-directories, and validates that the formats are in the list of allowed formats.